charts/seqr/values.yaml

```diff
@@ -156,18 +156,10 @@ volumeMounts: {}
 
 jobAfterHook: ""
 jobBeforeHook: ""
-cronJobs: {}
-# Examples
-# cronJobs:
-#  - name: sleep
-#    schedule: "5-59/5 * * * *"
-#    command: "/bin/sleep 2"
-#    timeout_s: 3600
-#  - name: fail_50
-#    schedule: "6-59/5 * * * *"
-#    command: "/bin/sleep 2 && exit $((RANDOM % 2))"
-
-checkForNewSamplesJobEnabled: false
+cronJobs:
+  - name: check-new-samples-job
+    schedule: "*/10 * * * *"
+    command: "python manage.py check_for_new_samples_from_pipeline"
```

Comment:
My read of the new code was that we don't need to pass in reference genome or dataset type any more, so `datasetVersions` is no longer necessary.